libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/route/RouteViewModel.java

```diff
@@ -95,14 +95,14 @@ public void fetchRouteNewOrigin(Point newOrigin) {
   private void fetchRoute(Point origin, Point destination) {
     if (origin != null && destination != null) {
       NavigationRoute.Builder routeBuilder = NavigationRoute.builder()
-        .accessToken(Mapbox.getAccessToken())
-        .destination(destination);
+        .accessToken(Mapbox.getAccessToken());
 
       if (locationHasBearing()) {
         fetchRouteWithBearing(routeBuilder, origin);
```

Comment:
The builder finishes building in `fetchRouteWithBearing` and `fetchRouteWithoutBearing`, so in this case, the destination will never be added.  Adding the destination in those methods would fix this

Reply:
ah good catch